### DISCUSSION TITLE
Advanced App Clip Experiences

### DISCUSSION THREAD

**Discussion by fawkeswei** [fastlane/fastlane#17766] [Ideas]
Will deliver support registering Advanced App Clip Experiences?

**Comment by joehinkle11** [comment]
I would love this feature. Here's some more information from Apple on how to setup experiences right now. It's a manual pain to do
https://help.apple.com/app-store-connect/#/dev5b665db74

**Comment by joehinkle11** [comment]
Another thing to note is that the App Store Connect API doesn't have a public endpoint to hit at this point in time
